backend/tests/baserow/contrib/builder/test_runtime_formula_results.py

```diff
@@ -0,0 +1,580 @@
+        user=user, table=table, type_name="text", name="Fruit (alternate)"
+    )
+    field_rating = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="number",
+        name="Rating",
+        number_decimal_places=2,
+    )
+    field_weight = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="number",
+        name="Weight KGs",
+        number_decimal_places=2,
+    )
+    field_weight_duplicate = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="number",
+        name="Weight KGs (duplicate)",
+        number_decimal_places=2,
+    )
+    field_weight_alternate = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="number",
+        name="Weight KGs (alternate)",
+        number_decimal_places=2,
+    )
+    field_harvested = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="date",
+        name="Harvested",
+        date_include_time=True,
+    )
+    field_harvested_duplicate = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="date",
+        name="Harvested (duplicate)",
+        date_include_time=True,
+    )
+    field_harvested_alternate = field_handler.create_field(
+        user=user,
+        table=table,
+        type_name="date",
+        name="Harvested (alternate)",
+        date_include_time=True,
+    )
+    field_tasty = field_handler.create_field(
+        user=user, table=table, type_name="boolean", name="Is Tasty"
+    )
+    field_tasty_duplicate = field_handler.create_field(
+        user=user, table=table, type_name="boolean", name="Is Tasty (duplicate)"
+    )
+    field_tasty_alternate = field_handler.create_field(
+        user=user, table=table, type_name="boolean", name="Is Tasty (alternate)"
+    )
+    field_even = field_handler.create_field(
+        user=user, table=table, type_name="number", name="Even number"
+    )
+    field_odd = field_handler.create_field(
+        user=user, table=table, type_name="number", name="Odd number"
+    )
+    field_datetime_format = field_handler.create_field(
+        user=user, table=table, type_name="text", name="Datetime Format"
+    )
+
+    fields = [
+        field_fruit,
+        field_fruit_duplicate,
+        field_fruit_alternate,
+        field_rating,
+        field_weight,
+        field_weight_duplicate,
+        field_weight_alternate,
+        field_harvested,
+        field_harvested_duplicate,
+        field_harvested_alternate,
+        field_tasty,
+        field_tasty_duplicate,
+        field_tasty_alternate,
+        field_even,
+        field_odd,
+        field_datetime_format,
+    ]
+
+    row_handler = RowHandler()
+    rows = [
+        row_handler.create_row(
+            user=user,
+            table=table,
+            values={
+                f"field_{fields[0].id}": row[0],
```

Comment:
Could you use `fields[0].db_column` here instead? (Some future refactor could be easier if we use that believe me 😅 ).

True everywhere you build the field name manually.